### DISCUSSION TITLE
Can't deploy my Next app on Vercel platform

### DISCUSSION THREAD

**Discussion by kvago36** [vercel/next.js#55435] [Help] [answered]
### Summary

Hi!

Im trying to deploy my Next app on Vercel, but every build fails with same error. Im using [next.cloudinary.dev](https://next.cloudinary.dev/) for image optimization 

// Image component
import { CldImage } from 'next-cloudinary';

Can someone please explain what this error means 
`You must supply a cloudName when initializing the asset` 

### Additional information

```js
You must supply a cloudName when initializing the asset
```

**Comment by icyJoseph** [comment, marked as answer]
I guess you need to set `NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME` or something like that in your Vercel Dashboard?

**Reply by kvago36** [reply]
Thank you! Added `NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME` to my Vercel Dashboard and build successfully deployed
